ui/src/core_components/CorePDF.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+	<div class="CorePDF">
+		<object class="pdf" :data="fields.source.value" type="application/pdf" width="100%" height="100%">
```

Comment:
Are these `width` and `height` properties necessary? They remind me of prehistoric HTML, haven't seen them in a while. I'm not familiar with `object` so maybe it's some strange thing that only applies to `object`?